src/components/set-image-card/index.ts

```diff
@@ -0,0 +1 @@
+export { SetImageCard } from "./SetImageCard";
```

Comment:
Please use DS image card component.

Reply:
@diegomart2000  this makes no sense to me since the the is not image card component used inside the set image card